pykern/pkcli/github.py

```diff
@@ -28,12 +28,15 @@
 _GITHUB_HOST = "github.com"
 _GITHUB_URI = "https://" + _GITHUB_HOST
 _GITHUB_API = "https://api." + _GITHUB_HOST
-_WIKI_ERROR_OK = (
-    r"fatal: repository 'https://github.com/radiasoft/.*.wiki.git/' not found"
-)
+_WIKI_ERROR_OK = r"fatal: repository 'https://github.com/[\w\-\.\_]+/[\w\-\.\_]+.wiki.git/' not found"
 _RE_TYPE = type(re.compile(""))
 _MAX_TRIES = 3
-_TEST_REPOS = ["test-pykern-github", "test-pykern-github-no-wiki"]
+_TEST_REPOS = [
+    ("radiasoft", "test-pykern-github"),
+    ("radiasoft", "test-pykern-github-no-wiki"),
+    ("biviosoftware", "home-env"),
```

Comment:
@moellep can you create an repo in biviosoftware? Please name it something like `test-pykern-github-no-wiki` and put just a README.md in the repo

Reply:
I've created https://github.com/biviosoftware/test-pykern-github-no-wiki now

Reply:
Added to test